src/utils/BrowserUtilities.ts

```diff
@@ -19,15 +19,26 @@ import { getUserAgentRegExp } from "browserslist-useragent-regexp";
  */
 
 import { ReaderConfig } from "../navigator/IFrameNavigator";
+import * as HTMLUtilities from "./HTMLUtilities";
 
 /** Returns the current width of the document. */
 export function getWidth(): number {
-  return document.documentElement.clientWidth;
+  const wrapper = HTMLUtilities.findRequiredElement(
+    document,
+    "#iframe-wrapper"
+  ) as HTMLDivElement;
+
```

Comment:
We should make the `findRequiredElement` generic:
```ts
const el = findRequiredElement<HTMLDivElement>(document, "#iframe-wrapper");
```
so that we don't have to keep casting these variables

Reply:
good idea.